src/lib.rs

```diff
@@ -1,7 +1,6 @@
-#![feature(const_trait_impl, const_mut_refs, const_fn_floating_point_arithmetic)]
+#![feature(const_trait_impl, const_mut_refs, const_fn_floating_point_arithmetic, effects)]
 
 mod impls;
-pub use impls::*;
```

Comment:
not actually needed and was complaining that imports are not used.

Reply:
i would like to leave this in, as this is needed to be compatible with the core / std implementation. (see README)
(Or am I wrong?)

Reply:
Not sure what you mean by compatible.
Can you elaborate?

![image](https://github.com/haennes/const_ops/assets/8925621/1d22790d-bf5d-4e7a-a75e-a09d9f1ddbd9)